### DISCUSSION TITLE
Toggle/uncomplete all tasks in file

### DISCUSSION THREAD

**Discussion by bellebethcooper** [notaapp/talk#333] [Feature Requests] [answered]
I would love to have an option like the "Toggle Task" option in the View menu right now that would "uncomplete" all tasks in the current file/note. I often make checklists in Markdown files that I use over and over again, but I tend to make tasks in those files and not toggle the tasks because it's such a hassle to go back and un-tick them when I'm done with the checklist.

**Comment by astoilkov** [comment, marked as answer]
Will ⌘A, hit Toggle Task (⌘.) twice work for you?

**Reply by bellebethcooper** [reply]
Ah, yep that works! Sorry, I didn't think to try this. I think I assumed it would only work when the cursor was on a line with a task in it. Thanks!
